src/Modal/Modal.Component.js

```diff
@@ -0,0 +1,255 @@
+            },
+            {
+              name: 'primaryBtnText',
+              description:
+                'String - Text to display on primary button in footer. Footer and buttons will not display unless "primaryBtnText" value is set.'
+            },
+            {
+              name: 'secondaryBtnText',
+              description:
+                'String - Text to display on secondary button in footer. Footer and buttons will not display unless "primaryBtnText" value is set.'
+            }
+          ]}
+        />
+
+        <Separator />
+
+        <h2>Informational Modal</h2>
+        <Description>
+          This is used to present information to the user but the Alert
+          Component doesn’t fit all the information.
+        </Description>
+        <DocsTile>
+          <button onClick={this.showHideModal}>Show Information Modal</button>
+          {this.state.bShowInfoModal ? (
+            <Modal title="Product Added" onClose={this.showHideModal}>
+              <div>
+                <b>The new product have been added to your catalog.</b>
+                <br />
+                <br />
+                Automatic Product ID: <b>PD-3465334</b>
+                <br />
+                <br />
+                Expiration date: <b>13/03/2018</b>
+                <br />
+                <br />
+              </div>
+            </Modal>
+          ) : (
+            ''
+          )}
+        </DocsTile>
+        <DocsText>{this.informationModalCode}</DocsText>
+        <Separator />
+
+        <h2>Confirmation Modal</h2>
+        <Description>
+          This is used to confirm with the user before continuing with a
+          destructive or complex action. In this case, the modal has action
+          buttons at the bottom.
+        </Description>
+        <DocsTile>
+          <button onClick={this.showHideConfirmModal}>
+            Show Confirmation Modal
+          </button>
+          {this.state.bShowComfirmModal ? (
+            <Modal
+              title="Delete"
+              onClose={this.showHideConfirmModal}
+              primaryBtnText="Sure"
+              secondaryBtnText="No Way"
+            >
+              <div>
+                Do you want to delete item <b>X</b>?
+              </div>
+            </Modal>
+          ) : (
+            ''
+          )}
+        </DocsTile>
+        <DocsText>{this.confirmationModalCode}</DocsText>
+        <Separator />
+
+        <h2>Form Modal</h2>
+        <Description>
+          This is used for short forms in order to collect information from the
+          user.
+        </Description>
+        <DocsTile>
+          <button onClick={this.showHideFormModal}>Show Form Modal</button>
+          {this.state.bShowFormModal ? (
+            <Modal
+              title="Invite user"
+              onClose={this.showHideFormModal}
+              primaryBtnText="Invite"
+              secondaryBtnText="Cancel"
```

Comment:
If we have 'actions' as props, we can then pass these actions like this:
`<Modal
              title="Invite user"
              onClose={this.showHideFormModal}
              actions={
                    <React.Fragment>
                         <Button type="secondary">Cancel</Button>
                         <Button>Invite</Button>
                      </React.Fragment>
                 }>`